### PR TITLE
Added test for Screenshot class

### LINKED ISSUE #81: Write a test for screenshots.py
Expand upon the idea in graphics.py

### PR DESCRIPTION
**Pull request by naslundx**
Fixes #81

### CODE REVIEW

**Comment by naslundx** [comment]
Fixes #84

File: tests/screenshot_test.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+from PIL import Image
+
+import webtraversallibrary as wtl
+from tests.graphics_test import equal_images
+from webtraversallibrary.screenshot import Screenshot
+
+ORIGINAL_DIR = Path("tests/data/")
+
+
+class FakeWebdriver:
```
Comment (redfungus) [inline tests/screenshot_test.py:30]: Very minor comment but maybe MockWebdriver would be a better name?

File: webtraversallibrary/screenshot.py [hunk 2]
```diff
@@ -76,22 +76,21 @@ def capture(cls, name: str, driver: WebDriver, scale: float = 1.0, max_page_heig
...
 
                 if shot_idx * viewport.height > page_height - viewport.height:
                     new_viewport_area = (
                         0,
-                        viewport.height - (page_height - shot_idx * viewport.height),
+                        0,
                         viewport.width,
-                        viewport.height,
+                        page_height - shot_idx * viewport.height,
                     )
```
Comment (redfungus) [inline webtraversallibrary/screenshot.py:94]: Just to mention:
I'm not sure what the changes here mean because I don't know the context.

**Review by redfungus — APPROVED** [review]

File: tests/screenshot_test.py (hunk 1 above)
Comment (naslundx) [reply tests/screenshot_test.py]: Renamed! :)

**Pull request merged**